client/src/pages/UserProfile.js

```diff
@@ -1,53 +1,278 @@
       </CardContent>
-      <CardActions>
-        <Button size="small">Learn More</Button>
-      </CardActions>
+    <form onSubmit={handleSubmitForm}> 
+    <Grid container spacing={3}>
+      <Grid item xs={12}>
+        <TextField
+          required
+          id="firstName"
+          name="firstName"
+          label="First name"
+          fullWidth
+          autoComplete="fname"
+          value={firstName}
+          onChange={event => {
+            setFirstName(event.target.value)
+          }}
+        />
+        </Grid>
+        <Grid item xs={12}>
+        <TextField
+          required
+          id="lastName"
+          name="lastName"
+          label="Last name"
+          fullWidth
+          autoComplete="lname"
+          value={lastName}
+          onChange={event => {
+            setLastName(event.target.value)
+          }}
+        />
+        </Grid>
+        <Grid item xs={12}>
+        <FormControl variant="outlined" className={classes.formControl}>
+        <InputLabel id="demo-simple-select-outlined">Gender</InputLabel>
+        <Select
+          id="demo-simple-select-outlined"
+          value={gender}
+          onChange={handleGenderChange}
+          label="Gender"
+        >
+          <MenuItem value={'male'}>Male</MenuItem>
+          <MenuItem value={'female'}>Female</MenuItem>
+          <MenuItem value={'other'}>Other</MenuItem>
+        </Select>
+        </FormControl>
+        </Grid>
+        <Grid item xs={12}>
+        <KeyboardDatePicker
+          margin="normal"
+          id="date-picker-dialog"
+          label="Birth Date"
+          format="MM/dd/yyyy"
+          value={selectedDate}
+          onChange={handleDateChange}
+          KeyboardButtonProps={{
+            'aria-label': 'change date',
+          }}
+        />
+        </Grid>
+      <Grid item xs={12}>
+        <TextField
+          required
+          id="email"
+          name="email"
+          label="Email Address"
+          fullWidth
+          autoComplete="email address"
+          value={emailAddress}
+          onChange={event => {
+            setEmailAddress(event.target.value)
+          }}
+        />
+      </Grid>
+      <Grid item xs={12} sm={6}>
+        <TextField
+          required
+          id="phoneNumber"
+          name="phoneNumber"
+          label="Phone Number"
+          fullWidth
+          autoComplete="phone number"
+          value={phoneNumber}
+          onChange={event => {
+            setPhoneNumber(event.target.value)
+          }}
+        />
+      </Grid>
+      <Grid item xs={12}>
+        <TextField
+          required
+          id="address1"
+          name="address1"
+          label="Address line 1"
+          fullWidth
+          autoComplete="billing address-line1"
+          value={addressOne}
+          onChange={event => {
+            setAddressOne(event.target.value)
+          }}
+        />
+      </Grid>
+      <Grid item xs={12}>
+        <TextField
```

Comment:
you can consider making these TextFields into its own component, taking in their respected props